docs-src/app-dev-context/customize-workflow-type.md

```diff
@@ -2,9 +2,11 @@
 id: customize-workflow-type
 title: How to customize your Workflow Type
 description: You can set a custom name for your Workflow Type.
-sidebar_label: Customize Workflow Type
+sidebar_label: Workflow Type
```

Comment:
We are adding "Customize" to the Activity Type one - why remove it here?

i.e -> Customize Activity Type
-> Customize Workflow Type

Reply:
Ya, I don't mind one way or the other here actually - just that we have the same for each one.